### ISSUE TITLE
NAA client on Outlook desktop classic throws an exception

### ISSUE THREAD

**Issue by dariussantaras** [OfficeDev/office-js#5210]
## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: 2412 Build 18324.20092
* Operating System: Windows
* Browser (if using Office on the web): N/A
 
## Expected behavior

The call to `createNestablePublicClientApplication` should return an instance of `IPublicClientApplication` without throwing an exception.

## Current behavior

`createNestablePublicClientApplication` throws an exception in Outlook Desktop classic:

```
TypeError: Cannot read properties of undefined (reading 'href') 
```

As far as I can tell, this only happens in event based activation

## Steps to reproduce

1. call `createNestablePublicClientApplication`

## Context

I can't use NAA to get a token on the old desktop client.

I have tried to track down the source of this error and found that it broke in `@azure/msal-browser` v3.27.0
I raised an issue on their tracker (https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/7451), but there has been no response.

## Useful logs

```
TypeError: Cannot read properties of undefined (reading 'href') 
```

**Comment by davidchesnut** [comment]
Hi @dariussantaras,

Looks like this is a regression in the msal-browser library. If you use version 3.26.1 of msal-browser it doesn't have the bug. The library will get updated soon with a fix so just keep an eye out for the next version.

Thanks,
David

**Comment by dariussantaras** [comment]
thank you, I'll keep an eye out

**Comment by davidchesnut** [comment]
Fixed in version 3.28.1 and later.
